providers/os/connection/container/image_connection.go

```diff
@@ -21,6 +22,12 @@ import (
 
 // NewImageConnection uses a container image reference as input and creates a tar connection
 func NewImageConnection(id uint32, conf *inventory.Config, asset *inventory.Asset, img v1.Image) (*tar.Connection, error) {
+	// FIXME: DEPRECATED, remove in v12.0 vv
+	// The SkipDiscovery flag should always be set from v12
```

Comment:
nit: The comment should mention `DelayDiscovery`, I think.

Reply:
fixed